From fa512b5353d151a79a2b7a93ce2b0bbc024cfc6f Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 10 May 2018 13:23:50 +0800
Subject: [PATCH 3763/5242] MLK-18265-1 ARM64: dts: freescale: imx8qm: remove
 GPIO clocks

commit  aa2b68e23d08bad15a2cf4c58b5853e26fb6a67f from
https://source.codeaurora.org/external/imx/linux-imx.git

All GPIOs clock will be kept ON in SCFW by default, here
remove all GPIOs management to make sure they are always
ON.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Tested-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |   14 --------------
 1 file changed, 14 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index a1b6936..4d379c3 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -2707,8 +2707,6 @@
                 power-domains = <&pd_lsio_gpio0>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		clocks = <&clk IMX8QM_GPIO0_IPG_S_CLK>;
-		clock-names = "ipg";
 	};
 
 	gpio1: gpio@5d090000 {
@@ -2720,8 +2718,6 @@
                 power-domains = <&pd_lsio_gpio1>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		clocks = <&clk IMX8QM_GPIO1_IPG_S_CLK>;
-		clock-names = "ipg";
 	};
 
 	gpio2: gpio@5d0a0000 {
@@ -2733,8 +2729,6 @@
                 power-domains = <&pd_lsio_gpio2>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		clocks = <&clk IMX8QM_GPIO2_IPG_S_CLK>;
-		clock-names = "ipg";
 	};
 
 	gpio3: gpio@5d0b0000 {
@@ -2746,8 +2740,6 @@
                 power-domains = <&pd_lsio_gpio3>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		clocks = <&clk IMX8QM_GPIO3_IPG_S_CLK>;
-		clock-names = "ipg";
 	};
 
 	gpio4: gpio@5d0c0000 {
@@ -2759,8 +2751,6 @@
                 power-domains = <&pd_lsio_gpio4>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		clocks = <&clk IMX8QM_GPIO4_IPG_S_CLK>;
-		clock-names = "ipg";
 	};
 
 	gpio5: gpio@5d0d0000 {
@@ -2783,8 +2773,6 @@
                 power-domains = <&pd_lsio_gpio6>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		clocks = <&clk IMX8QM_GPIO6_IPG_S_CLK>;
-		clock-names = "ipg";
 	};
 
 	gpio7: gpio@5d0f0000 {
@@ -2796,8 +2784,6 @@
                 power-domains = <&pd_lsio_gpio7>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
-		clocks = <&clk IMX8QM_GPIO7_IPG_S_CLK>;
-		clock-names = "ipg";
 	};
 
 	gpio0_mipi_csi0: gpio@58222000 {
-- 
1.7.9.5

